doc/devel/contributing.rst

```diff
@@ -19,6 +19,59 @@ is expected to abide by our
 Below, you can find a number of ways to contribute, and how to connect with the
 Matplotlib community.
 
+.. _start-contributing:
+
+Getting started with contributing?
+==================================
+
+There is no pre-defined pathway for new contributors; you can look at existing
+issue and pull request discussions, and follow the discussions during code
+review. Or you can deep-dive into a subset of code-base to understand what is
+going on.
+
+There are a few typical new contributor profiles:
+
+* **You are a Matplotlib user, and you see a bug, a potential improvement, or
+  something that annoys you, and you can fix it.**
+
+  This is how a lot of people approach open-source development. You can search
+  our issue tracker for an existing issue that describes your problem or open a
+  new issue to inform us of the problem you observed and discuss the best
+  approach to fix it. If your contributions would not be captured on GitHub
+  (social media, communication, educational content), you can also reach out to
+  us on gitter_, `Discourse <https://discourse.matplotlib.org/>`__ or attend any
+  of our `community meetings <https://scientific-python.org/calendars>`__.
+
+* **You are not a regular Matplotlib user but a domain expert: you know about
+  visualization, 3D plotting, design, technical writing, statistics, or some
+  other field where Matplotlib could be improved.**
+
+  In this case, even if you need help with programming skills or don't know
+  where to start, you have a focus on a specific application and domain and can
+  start there. In this case, maintainers can help you figure out the best
+  implementation; you can open an issue or pull request with a starting point,
+  and we'll be happy to discuss technical approaches. If you prefer, you can use the
+  `GitHub functionality for "draft" pull requests <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft>`__
+  and request early feedback on whatever you are working on.
```

Comment:
My impression is that it is much easier to get feedback if it is not in draft mode... But I agree that it would be nice if it worked as above.

Reply:
Yeah, I think that is fair. How about this version?